### ISSUE TITLE
`;!zprint {:format :off}` deletes the outer key from namespaced maps 

### ISSUE THREAD

**Issue by miridius** [kkinnear/zprint#275]
Running zprint on this:
```clojure
;!zprint {:format :off}
#::foo{:bar :baz}
```
Outputs this:
```clojure
;!zprint {:format :off}
{:bar :baz}
```

The same thing happens with `;!zprint {:format :skip}`

**Comment by kkinnear** [comment]
That's even worse!  Thanks again for noticing this and reporting it.  This is in the same general place as Issue #274, and I've reproduced and, again, fixed it I think.  More testing required here too.  This will also be fixed in `1.2.5`.

For what it is worth, this used to work -- I broke it in `1.2.4`.  It works in `1.2.3`, so if this problem is getting in your way you could probably back off to release `1.2.3` and do fine (unless you need something else from `1.2.4`, of course).  The same can't be said for the indentation bug, #274.  That has been broken for a good long time.

Also, thank you for boiling down both of these problems into small, easily reproducible examples.  I makes my job a lot easier -- which is why I was able to not only reproduce these problems but prototype a fix in a few minutes after reproducing it.  It takes extra work on your part to do that, and I certainly appreciate it!

Thanks much!

**Comment by miridius** [comment]
Sure, no problem @kkinnear! I should be the one thanking you, not the other way around - you're the one doing the actual hard work here 😊

**Comment by kkinnear** [comment]
This is fixed in `1.2.5`.
